python/cudf_kafka/cudf_kafka/_lib/kafka.pyx

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+from libcpp.string cimport string
+from libcpp.map cimport map
+from libc.stdint cimport int64_t
+from cudf._lib.cpp.io.types cimport datasource
+from libcpp.memory cimport unique_ptr, make_unique
+from cudf_kafka._lib.kafka cimport kafka_consumer
+
+cdef class KafkaDatasource(Datasource):
+
+    def __init__(
+        self,
+        kafka_configs,
+        topic,
+        partition,
+        start_offset,
+        end_offset,
+        batch_timeout,
+        delimiter,
+    ):
+
+        self.kafka_configs = kafka_configs
+        self.topic = topic
+        self.partition = partition
+        self.start_offset = start_offset
+        self.end_offset = end_offset
+        self.batch_timeout = batch_timeout
+        self.delimiter = delimiter
```

Comment:
Given these all map directly into c attributes, any reason to not just make them public c attributes and initialize them in the `__cinit__`?

Reply:
nope. I will do that